inc/spbc-settings.php

```diff
@@ -2519,7 +2519,8 @@ function spbc_field_traffic_control_log()
         echo '</div>';
         return;
     }
-
+    $autoblock_amount = isset($spbc->settings['traffic_control__autoblock_amount']) ? $spbc->settings['traffic_control__autoblock_amount'] : 1000;
```

Comment:
Please, use `$spbct->default_settings['traffic_control__autoblock_amount']` instead of hardcoded value.

Reply:
ok